src/mars.d

```diff
@@ -1942,4 +1942,25 @@ private void addDefaultVersionIdentifiers()
     if (global.params.useArrayBounds == BOUNDSCHECKoff)
         VersionCondition.addPredefinedGlobalIdent("D_NoBoundsChecks");
     VersionCondition.addPredefinedGlobalIdent("D_HardFloat");
+
+    printPredefinedVersions();
+}
+
+private void printPredefinedVersions()
+{
+    import core.stdc.stdio;
+    if (global.params.verbose && global.params.versionids) {
+        printf("predefs  ");
+        int col = 10;
+        for (int i = 0; i < global.params.versionids.dim; i++) {
+            int len = cast(int) strlen(global.params.versionids.data[i]);
+            if (col + len > 80) {
+                col = 10;
+                printf("\n         ");
+            }
+            col += len;
+            printf(" %s", global.params.versionids.data[i]);
+        }
+        printf("\n");
+    }
```

Comment:
I did change the formatting so it's all done on 1 line and that newline is the simplest way to assure formatting.